src/Resources/scaffolds/6.0/change-password/tests/Functional/User/ChangePasswordTest.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+namespace App\Tests\Functional\User;
+
+use App\Factory\UserFactory;
+use App\Tests\Browser\Authentication;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Zenstruck\Browser\Test\HasBrowser;
+use Zenstruck\Foundry\Test\Factories;
+use Zenstruck\Foundry\Test\ResetDatabase;
+
+/**
+ * @author Kevin Bond <kevinbond@gmail.com>
+ */
+final class ChangePasswordTest extends KernelTestCase
+{
+    use HasBrowser, Factories, ResetDatabase;
```

Comment:
This is another test that's GREAT... but maybe too great (big) for giving to users :)